opi/__init__.py

```diff
@@ -487,12 +485,12 @@ def ask_for_option(options, question="Pick a number (0 to quit):", option_filter
 		If needed, a pager will be used, unless disable_pager is True.
 	"""
 
-	padding_len = math.floor(math.log10(len(options)))
+	padding_len = len(str(len(options))) + 1
```

Comment:
```suggestion
	padding_len = len(str(len(options)))
```

Sorry for the confusion - I somehow compared this wrong the first time.

```py
>>> options = ['foo']*37
>>> options
['foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo', 'foo']
>>> math.floor(math.log10(len(options)))
1
>>> len(str(len(options)))
2
```

So it seems you were actually right by dropping the `+1`